src/localplayer.cpp

```diff
@@ -327,6 +327,7 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 			if (is_first || len < distance) {
 				m_standing_node = colinfo.node_p;
 				distance = len;
+				is_first = false;
```

Comment:
Bugfix?

Reply:
Yes, but ingame it doesn't matter too much because `move()` is called multiple times per dtime, whereas the second call corrected this issue.